kotlin/services/support/Readme.md

```diff
@@ -0,0 +1,72 @@
+- [Describe an attachment](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/support/src/main/kotlin/com/example/support/SupportScenario.kt) (describeAttachment command)
+- [Describe services](https://github.com/awsdocs/aws-doc-sdk-examples/blob/kotlin/services/support/src/main/kotlin/com/example/support/SupportScenario.kt) (describeServices command)
+- [Describe severity levels](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/support/src/main/kotlin/com/example/support/SupportScenario.kt) (describeSeverityLevels command)
+- [Describe cases](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/support/src/main/kotlin/com/example/support/SupportScenario.kt) (describeCases command)
+- [Describe communications](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/support/src/main/kotlin/com/example/support/SupportScenario.kt) (describeCommunications command)
+- [Resolve the case](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/support/src/main/kotlin/com/example/support/SupportScenario.kt) (resolveCase command)
+
+### Scenarios 
+
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+
+- [Get started with cases](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/support/src/main/kotlin/com/example/support/SupportScenario.kt) (Multiple commands)
+
+## Run the AWS Support Kotlin file
+
+**Be very careful** when running an operation that deletes or modifies AWS resources in your account. We recommend creating separate test-only resources when experimenting with these examples.
+
+To run these examples, set up your development environment. For more information, 
+see [Get started with the SDK for Kotlin](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/get-started.html). 
+
+
+ ## Test the AWS Support Kotlin file
+
+You can test the Kotlin code example for AWS Support by running a test file named **SupportTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/kotlin** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
+
+You can run the JUnit tests from an IDE, such as IntelliJ, or from the command line. As each test runs, you can view messages that inform you if the various tests succeed or fail. For example, the following message informs you that Test 3 passed.
+
+	Test 3 passed
+
+**WARNING**: _Running these JUnit tests manipulates real AWS resources and might incur charges on your account._
+
+ ### Properties file
+Before running the JUnit tests, you must define values in the **config.proerties** file located in the **resources** folder. This file contains values that are required to run the JUnit tests. If you do not define all values, the JUnit tests fail.
```

Comment:
Should this be config.properties (typo?)

Reply:
fixed